### DISCUSSION TITLE
termux official creat  codeberg

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [termux/termux-packages#8660] [Ideas]
link site codeberg.org 
repo repository

**Comment by ghost** [comment]
You can mirror the Termux repository to another git hosting but the focus is Github so Termux maintainers won't bother accepting issues and pull/merge requests in codeberg

**Reply by ExtinctDrift19** [reply]
Okay

**Reply by ghost** [reply]
Exactly. Switching Git hosting wouldn't be easy for us. Specifically for purpose of "not tracking" solution it is useless and will rather introduce issues rather than resolve. Termux doesn't have own infrastructure and relies on third parties.

**Reply by finagolfin** [reply]
Why would it be difficult? The whole point of git is that we have all the past source changes and commit log. There are easy importers to get all the past PR and issue comments from this site. It should be pretty easy to move, maybe some CI script changes and that's it.

I think we should seriously consider moving to gitlab or something else, but I don't really have a problem with github right now and have not explored the alternatives.
